content/docs/conditional-rendering.md

```diff
@@ -44,13 +44,14 @@ ReactDOM.render(
 
 [**Try it on CodePen**](https://codepen.io/gaearon/pen/ZpVxNq?editors=0011)
 
-This example renders a different greeting depending on the value of `isLoggedIn` prop.
+यह उदहारण `isLoggedIn` प्रोप के वैल्यू के अनुसार अलग अलग ग्रीटिंग्स रेंडर करता हे।
 
-### Element Variables {#element-variables}
+### एलिमेंट वेरिएबल {#element-variables}
 
-You can use variables to store elements. This can help you conditionally render a part of the component while the rest of the output doesn't change.
+आप एलिमेंट्स को स्टोर करने के लिए वेरिएबल्स का उसे कर सकते हो। 
+जिससे आपको कंडीशनल रेंडरिंग में मदद होगी और बाकि का आउटपुट चेंज नहीं होगा। 
```

Comment:
Remove this line break. Change line 51-52 to this
```
आप एलिमेंट्स को स्टोर करने के लिए वेरिएबल्स का उसे कर सकते हो। जिससे आपको कंडीशनल रेंडरिंग में मदद होगी और बाकि का आउटपुट चेंज नहीं होगा। 
```
both sentences in single line same like english

Reply:
done